### PR TITLE
(feat) support for markdown templates

### PR DESCRIPTION
**Pull request by dselman**
Signed-off-by: Dan Selman <danscode@selman.org>

# Issue #404
If a template contains a `template.md` file then it is used instead of `template.tem` to generate the parser.

### Changes
- Load the `template.md` file using the CommonMarkParser to generate an AST
- Convert the AST to a markdown string
- Generate the parser from the markdown string
- Added tests for latedeliveryandpenalty and copyright-license

### Flags
- We may want to consider rewriting the parser generator to use the AST, rather than the text "chunks"

### CODE REVIEW

File: packages/cicero-core/src/grammars/datetime.js [hunk 1]
```diff
@@ -95,6 +95,9 @@ YYYY ->[0-9] [0-9] [0-9] [0-9]
 # Timezone Offset
 Z -> ("+"|"-") [0-9] [0-9] ":" [0-9] [0-9]
 {% (d) => {return d[0] + d[1] + d[2] + ":" + d[4] + d[5]}%}
+
```
Comment (jeromesimeon) [inline packages/cicero-core/src/grammars/datetime.js]: What is this change about?

Comment (dselman) [reply packages/cicero-core/src/grammars/datetime.js]: When we generate the grammar for a `DateTime` fields we refer to `DateTime` rule. While testing I noticed that we were getting Nearley warnings for references to this missing rule.

E.g. I think without this the parser would be broken if you had a concept in your model that had a field of type `DateTime` and then you used that concept in your contract model.

Comment (jeromesimeon) [reply packages/cicero-core/src/grammars/datetime.js]: sounds unrelated to the PR, but a possible bug fix? I'm wondering if this is related to the change that was made to address: https://github.com/accordproject/cicero/issues/375

Comment (jeromesimeon) [reply packages/cicero-core/src/grammars/datetime.js]: it would be useful to have a clear issue open on that.

Comment (dselman) [reply packages/cicero-core/src/grammars/datetime.js]: Done: https://github.com/accordproject/cicero/issues/406

**Pull request merged**